EnglishFilter/sections/general_extensions.txt

```diff
@@ -232,6 +232,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/75451
+moviemaniac.org#%#//scriptlet("set-constant", "po", "true")
```

Comment:
@wout-codes 
This rule is not necessary, ||moviemaniac.org/asset/bann.js
 is enough

Reply:
What if they change their file name? There is no harm in making the other rule stronger.

Reply:
And in my opinion, I think Scriptlets are way better than blocking the whole file since the file name can be changed in any seconds.

Reply:
@wout-codes 
They also may change the name of variable and anything else but it doesn't make sense to add redundant rules for all possible changes

Reply:
Okay then.